packages/webview_flutter/example/test_driver/webview_flutter_e2e.dart

```diff
@@ -536,6 +536,35 @@ void main() {
     expect(title, 'Some title');
   });
 
+  testWidgets('setInitialScale', (WidgetTester tester) async {
+    final Completer<WebViewController> controllerCompleter =
+        Completer<WebViewController>();
+
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: WebView(
+          key: GlobalKey(),
+          initialUrl: 'https://flutter.dev/',
+          onWebViewCreated: (WebViewController controller) {
+            controllerCompleter.complete(controller);
+          },
+        ),
+      ),
+    );
+
+    final WebViewController controller = await controllerCompleter.future;
+    expect(controller.setInitialScale(50), completes);
+    expect(
+      () => controller.setInitialScale(-1),
+      throwsA(isA<AssertionError>()),
+    );
+    expect(
+      () => controller.setInitialScale(150),
+      throwsA(isA<AssertionError>()),
+    );
```

Comment:
Can we test that the value was actually set? Would it affect the value returned by [onScaleChanged](https://developer.android.com/reference/android/webkit/WebViewClient.html#onScaleChanged(android.webkit.WebView,%20float,%20float)) ?